WireAnalytics/.swiftpm/WireAnalytics.xctestplan

```diff
@@ -0,0 +1,24 @@
+{
+  "configurations" : [
+    {
+      "id" : "1AD1FA32-0D55-4D7B-9E4B-C64035D0D3B3",
+      "name" : "Test Scheme Action",
+      "options" : {
+
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "codeCoverage" : false
```

Comment:
Did you disable code coverage explicitly?

Reply:
Done: https://github.com/wireapp/wire-ios/commit/ff908d23c2fcb4bc0d3a7333190c991ba2b013aa